charts/seqr/templates/check-for-new-samples-from-pipeline.yaml

```diff
@@ -55,5 +55,9 @@ spec:
       {{- with $.Values.deployment_sidecars }}
         {{- tpl . $ | nindent 8 }}
       {{- end }}
+      {{- with $.Values.nodeSelector }}
+      nodeSelector:
```

Comment:
I just saw this missing and added.  I'm not sure if we've ever seen this workload assigned off default-pool, but certainly a possibility.